.github/workflows/android.yaml

```diff
@@ -294,6 +304,9 @@ jobs:
     - name: Check Licenses for modules
       working-directory: ./platform/jvm
       run: ./gradlew replay:checkLicense common:checkLicense capture:checkLicense
+    - name: Set up local.properties, required for cargo-ndk
```

Comment:
weird, do they require any vars to be set there?

Reply:
doesn't seem like it https://github.com/willir/cargo-ndk-android-gradle/issues/23